### PR TITLE
Add Site Reliability Engineering Roles 

### PR DESCRIPTION
**Pull request by paulolai**
Octopus Deploy needs engineers who know enough about both software and systems engineer to build and maintain our internal tools and infrastructure. This job ladder uses our software engineering ladder as a base with some targeted changes to shift the focus to reliable systems.

### CODE REVIEW

File: Site-Reliability-Engineering/L3-Senior-Site-Reliability-Engineer.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+- I ship ideas from conception to production.
+- I advocate for customer needs and motivations.
+- I decompose larger requirements into smaller, more manageable pieces, to deliver functionality in an incremental and iterative manner.
+- I build requirements and technical recommendations based on the real needs of customers.
+
+<details>
+<summary>Examples</summary>
+
+- I led a shaped pitch from idea to production
```
Comment (cailyoung) [inline Site-Reliability-Engineering/L3-Senior-Site-Reliability-Engineer.md:77]: I'm not sure `shaped pitch` is the right term any more.

File: Site-Reliability-Engineering/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+# What are SREs?
+SREs understand software systems are not just a pile of code. They understand its purpose, the 
+machinery used to run it, how it\'s managed and that reliable operations require humans to 
+collaborate rather than individual heroism.
+
+They help teams to make the right reliability trade offs. They understand how to measure how well
+ a system is running, identify the high leverage points and move them to improve the outcomes, 
+ costs and/or manage risks. 
+
+They are our resident experts in learning, both from what works and doesn\'t, are unphased
```
Comment (cailyoung) [inline Site-Reliability-Engineering/README.md:35]: nitpick:
```suggestion
They are our resident experts in learning, both from what works and doesn\'t, are unfazed 
```

File: Site-Reliability-Engineering/L2-Site-Reliability-Engineer.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+## 🦉 Domain expertise
+
+- I am competent in my domain.
+- I am able to work independently on small projects.
+- I identify and raise potential problems with my peers or lead before getting blocked.
+
+<details>
+<summary>Examples</summary>
+
+- I completed maintenance work in a complex area of our systems, relying on my own knowledge and
```
Comment (octoliamp) [inline Site-Reliability-Engineering/L2-Site-Reliability-Engineer.md:19]: can we adjust please to "relying on my own knowledge of networking and systems"

Comment (octoliamp) [reply Site-Reliability-Engineering/L2-Site-Reliability-Engineer.md:19]: as above, I think specific mention of networking, infrastructure and systems is needed here. This gives more focus on the specifics of SRE, compared to SWE.

File: Site-Reliability-Engineering/L3-Senior-Site-Reliability-Engineer.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+- I am advanced in my team's domain.
+- I can confidently step outside my own comfort zone and adapt quickly to new technologies.
+- I am able to codify conventions (where appropriate) into my team\'s systems
+- I make appropriate scaling, reliability, and maintenance trade-offs as they occur in practice.
+- I identify, qualify and reduce technical debt as appropriate to my team's priorities.
+
+<details>
+<summary>Examples</summary>
+
+- I set up or maintained an automated integration and delivery pipeline.
```
Comment (octoliamp) [inline Site-Reliability-Engineering/L3-Senior-Site-Reliability-Engineer.md:21]: suggest add "improved" to setup or maintain.

File: Site-Reliability-Engineering/L3-Senior-Site-Reliability-Engineer.md (hunk 1 above)
Comment (octoliamp) [reply Site-Reliability-Engineering/L3-Senior-Site-Reliability-Engineer.md:77]: I am ok with shaped pitch for the moment. A lot of our confluence info for R&D still mentions it as a tool we can use, even if we don't have formal ShapeUp cycles etc..

**Review by octoliamp — CHANGES_REQUESTED** [review]
comments added @paulolai can you make some small changes.

File: Site-Reliability-Engineering/L2-Site-Reliability-Engineer.md (hunk 3 above)
Comment (paulolai) [reply Site-Reliability-Engineering/L2-Site-Reliability-Engineer.md:19]: Thanks, I've updated

File: Site-Reliability-Engineering/L3-Senior-Site-Reliability-Engineer.md (hunk 4 above)
Comment (paulolai) [reply Site-Reliability-Engineering/L3-Senior-Site-Reliability-Engineer.md:21]: Done

File: Site-Reliability-Engineering/L3-Senior-Site-Reliability-Engineer.md (hunk 1 above)
Comment (paulolai) [reply Site-Reliability-Engineering/L3-Senior-Site-Reliability-Engineer.md:77]: Yes, let's leave it for now until we decide as a group what the new terms are.

**Review by octoliamp — APPROVED** [review]
happy with this now. Lets merge

File: Site-Reliability-Engineering/README.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+SREs understand software systems are not just a pile of code. They understand its purpose, the 
+machinery used to run it, how it\'s managed and that reliable operations require humans to 
+collaborate rather than individual heroism.
+
+They help teams to make the right reliability trade offs. They understand how to measure how well
+ a system is running, identify the high leverage points and move them to improve the outcomes, 
+ costs and/or manage risks. 
+
+They are our resident experts in learning, both from what works and doesn\'t, are unfazed 
+by chaos, they will calmly and practically get problems fixed.
```
Comment (kengel100) [inline Site-Reliability-Engineering/README.md:36]: Liam proposed some rewording of this sentence and another one below. All looks great, once those changes are made. 👍

**Review by kengel100 — APPROVED** [review]
LGTM including the minor changes that we discussed

File: Site-Reliability-Engineering/README.md (hunk 5 above)
Comment (paulolai) [reply Site-Reliability-Engineering/README.md]: How does this look now?

Comment (kengel100) [reply Site-Reliability-Engineering/README.md]: LGTM

**Review by octoliamp — APPROVED** [review]

**Comment by octoliamp** [comment]
ready to merge, pending CE and SRE line up with Kim and chat with Roy

**Pull request merged**